fetch-key-vault-secrets/README.md

```diff
@@ -0,0 +1,32 @@
+# Fetch Key Vault Secrets
+
+Fetch Secrets from the Azure Key vault taking Key Vault name and secret names as input.
+
+## Inputs
+- `KEY_VAULT`: Name of the Key Vault containing the secrets (Required)
+- `SECRETS`: Comma seperated secret names that are going to be fetched (Required)
+
+## Outputs
+- `SECRET_<secretname>`: Masked Value of the secret
```

Comment:
```suggestion
- `SECRET_<secretname>`: Value of the secret
```

The masked value would be just `x`s I think, whereas this should contain the real value.